swift/vapor-framework/Package.swift

```diff
@@ -0,0 +1,25 @@
+// swift-tools-version:5.2
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "server",
+    platforms: [
+        .macOS("10.15")
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        .package(url: "https://github.com/vapor/vapor.git", from: "4.3.0")
```

Comment:
could you use `.upToNextMinor(from: "4.3.0")` instead of `from: "4.3.0" ?
the purpose is to us the most up to date in **4.3.x**

Reply:
Done!